src/components/SignUp/AdditionalSkills.tsx

```diff
@@ -1,23 +1,224 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
+import Form from 'react-bootstrap/Form';
+import Checkbox from '../../genericComponents/Checkbox';
 import { Tagline, Title } from '../layout/Titles';
+import { colors, fonts } from '../../theme/styleVars';
+
+const skillCheckboxes = ['Dancing', 'Singing'];
 
 const AdditionalSkills: React.FC<{
   setForm: any;
   formData: any;
 }> = props => {
+  const { formData, setForm } = props;
+
+  const { additionalSkillsCheckboxes } = formData;
+
+  const isAdditionalSkillsCheckboxes = (skillOption: string) =>
+    additionalSkillsCheckboxes.indexOf(skillOption) > -1;
+
+  const skillOptionChange = (checkValue: any, skill: string) => {
+    let newSkills = [...additionalSkillsCheckboxes];
+
+    if (checkValue) {
+      // check value
+      if (newSkills.indexOf(skill) < 0) {
+        newSkills.push(skill);
+      }
+    } else {
+      // uncheck value
+      newSkills = newSkills.filter(sO => sO !== skill);
+    }
+
+    const target = {
+      name: 'additionalSkillsCheckboxes',
+      value: newSkills
+    };
+
+    setForm({ target });
+  };
+
+  const [input, setInput] = useState('');
```

Comment:
Minor organizational thing: I'd keep all state value things together at the top of the component after the formData lines (starting on line 20 or 21)